src/main/java/com/testsigma/sdk/RestAction.java

```diff
@@ -0,0 +1,4 @@
+package com.testsigma.sdk;
+
+public abstract class RestAction extends DriverAction{
```

Comment:
@SunilGembali Is this reviewed with Product team.. I feel that We can call this as APIAction. RestAction looks somewhat weird and hard to understand. Please check with them and refactor if needed

Reply:
@rajesh-testsigma  suggested to make it as RestApiAction. WIll change